service/src/main/kotlin/fi/espoo/evaka/varda/integration/MockVardaIntegrationEndpoint.kt

```diff
@@ -32,177 +36,226 @@ private val logger = KotlinLogging.logger {}
 @RestController
 @RequestMapping("/mock-integration/varda/api")
 class MockVardaIntegrationEndpoint(private val mapper: ObjectMapper) {
-    val decisions = mutableListOf<VardaDecision>()
-    val feeData = mutableListOf<VardaFeeData>()
-    val children = mutableListOf<VardaChildRequest>()
+    private val mutex = Mutex()
```

Comment:
kotlinx has been removed in master, so you should use standard library `ReentrantLock` instead.

See https://github.com/espoon-voltti/evaka/pull/272/commits/e6e9037085047b26a016f77f64f2ed42ec69acd2 for more info

Reply:
👍